### ISSUE TITLE
Installer Issues with Docker

### ISSUE THREAD

**Issue by unspecific** [InfoSecInnovations/concierge#10]
First fresh install (Mint Linux 20.3).

Have GPU so chose GPU.  Failed to start up with GPU.  Rerun the installer and use defaults.  Can't start because a container with the name already exists. Try the option to remove the containers at the beginning of the installer.  Says it works, but still can't start.  Delete Manually and the script errors our because the /opt/concierge/volumes exists but is empty (I think).

Solution was to go kill all dockers containers. (I didn't have any others)  I don't know docker.
```
docker stop $(docker ps -a -q)
docker rm $(docker ps -a -q)
```
or
```
docker rmi $(docker images -a -q)
```
Followed by 
```
docker volume prune
```

Term capture during the first install
https://pastebin.com/0HHKfNJa

Then when I tried again it kept erroring out on checking /opt/concierge/volumes after it was cleaned up.
https://pastebin.com/zZtbU6Qg

When I finally got it to work
https://pastebin.com/myStU85g

The install script needs some more error handling.

**Comment by besimorhino** [comment]
Thanks for the info! We absolutely do need more error handling -- "it works on our systems" is not a good stance... so thanks.

We'll review these and start working on fixes.

The docker cleanup stuff you're doing is a bit agressive though... other folks would be upset if we nuked containers & images they need for other things.

**Comment by unspecific** [comment]
Absolutely.  That is why I said I was not using Docker for anything else. It was an easy "restart" option.  

Yeah, I know it is alpha, so just pointing it out to help.
